src/App.jsx

```diff
@@ -1,33 +1,54 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+const filterBy = (query, setMovies) => {
+  const newMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query.trim().toLowerCase())
+      || movie.description.toLowerCase().includes(query.trim().toLowerCase()),
```

Comment:
```suggestion
const filterBy = (query, setMovies) => {
 const normalizedQuery = query.trim().toLowerCase();
 
  const newMovies = moviesFromServer.filter(
    movie => movie.title.toLowerCase().includes(normalizedQuery)
      || movie.description.toLowerCase().includes(normalizedQuery),
```
suggest to not repeat same operation twice